cmake/multilib.yaml.in

```diff
@@ -148,9 +148,18 @@ Mappings:
 # of each other, in the obvious way, but we don't have any libraries
 # for those at present, so there's no need to generate all their
 # flags.)
-- Match: --target=armv(8\.[1-9]|9|9\.[1-9])a-unknown-none-eabi
+- Match: --target=armv(8|8\.[1-9]|9|9\.[1-9])a-unknown-none-eabi
```

Comment:
You can use `(arm|thumb)` in the triple (see `armv7ve` above for example) to halve the number of `Match` blocks here, because the ISA doesn't matter for library selection.